addons/binding/org.openhab.binding.russound/src/main/java/org/openhab/binding/russound/internal/rio/RioConstants.java

```diff
@@ -19,118 +19,134 @@
 public class RioConstants {
 
     // BRIDGE TYPE IDS
-    public static final ThingTypeUID BRIDGE_TYPE_RIO = new ThingTypeUID(RussoundBindingConstants.BINDING_ID, "rio");
-    public static final ThingTypeUID BRIDGE_TYPE_CONTROLLER = new ThingTypeUID(RussoundBindingConstants.BINDING_ID,
+    public final static ThingTypeUID BRIDGE_TYPE_RIO = new ThingTypeUID(RussoundBindingConstants.BINDING_ID, "rio");
```

Comment:
Ah - I see what you meant about the lines - don't know why they differ (probably copy/pasted and fixed one without the other).  At any rate, I've updated to these latest comments and pushed.

Note: I've known about the requireNotNull and use it in current (other) bindings - but I didn't think about it when I made all these mods and tell you the truth - will probably leave it for a future date at this point (want to get this off my plate to get back to the sony binding)